runtime/src/main/cpp/StdCppStubs.cpp

```diff
@@ -32,7 +32,8 @@ RUNTIME_USED RUNTIME_WEAK extern "C" char* Konan_cxa_demangle(
 
 namespace std {
 RUNTIME_WEAK void __throw_length_error(const char* __s __attribute__((unused))) {
-  RuntimeAssert(false, __s);
+  // TODO: Don't we always need to abort here?
```

Comment:
I guess we do. Probably this code was written when assertions were always enabled unconditionally. Could you please fix this?